### PR TITLE
CART-154 added popup

### PR DESCRIPTION
**Pull request by RomanFedytskyi**

### CODE REVIEW

**Comment by dmitrij-borchuk** [comment]
Please see screenshot http://prntscr.com/ct4hs3

**Comment by dmitrij-borchuk** [comment]
I tried to add decal
See screenshot http://prntscr.com/ct4ior

File: dev/publicDev/decal/decalAdd.jade [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+      form.decal-form(name="decalForm"
+        ng-submit="vm.addDecal(decalForm)",
+        novalidate)
+          div.radio
+            span.radio-title Color of decal
+            input(type="radio",
+              name="color",
+              value="white",
+              ng-model="vm.decal.decalColor")
+            span.color White
```
Comment (dmitrij-borchuk) [inline dev/publicDev/decal/decalAdd.jade]: Click on label should set the radio batton

**Review by dmitrij-borchuk — CHANGES_REQUESTED** [review]

File: dev/publicDev/decal/decalAdd.jade (hunk 1 above)
Comment (RomanFedytskyi) [reply dev/publicDev/decal/decalAdd.jade]: fixed

**Review by vetalcore — APPROVED** [review]
please resolve conflicts with dev, (if all issues are fixed)

**Review by dmitrij-borchuk — APPROVED** [review]

**Pull request merged**